components/projects/NewProject/NewProjectForm.tsx

```diff
@@ -29,9 +30,32 @@ function NewProjectForm({ employees, customers }: NewProjectFormProps): JSX.Elem
         }
     };
 
-    const handleSumbit = (e: React.FormEvent<HTMLFormElement>) => {
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
         e.preventDefault();
-        console.log(state);
+        // console.log(state);
+        const data = {
+            id: state.id,
+            name: state.name,
+            description: state.description,
+            startDate: state.startDate,
+            endData: state.endDate,
+            customer: { id: state.customer?.id, name: state.customer?.name },
+            managingEmployee: {
+                id: state.managingEmployee?.id,
+                first_name: state.managingEmployee?.first_name,
+                last_name: state.managingEmployee?.last_name,
+                email: state.managingEmployee?.email,
+            },
+        };
+        const url = `${process.env.NEXT_PUBLIC_API_URL}/project`;
```

Comment:
`${process.env.NEXT_PUBLIC_API_URL}` should be a high level private const definition. Having it here in the middle of the method makes the code hard to read

Reply:
Fixed in bdfadda